x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/MetadataAttribute.java

```diff
@@ -25,14 +21,14 @@ public class MetadataAttribute extends TypedAttribute {
 
     private static final Map<String, Tuple<DataType, Boolean>> ATTRIBUTES_MAP = Map.of(
         "_version",
-        tuple(DataTypes.LONG, true),
+        tuple(DataTypes.LONG, false), // _version field is not searchable
         "_index",
         tuple(DataTypes.KEYWORD, true),
         "_id",
-        tuple(DataTypes.KEYWORD, false)
+        tuple(DataTypes.KEYWORD, false) // actually searchable, but fielddata access on the _id field is disallowed by default
```

Comment:
The `_id` could be pushed down, but since by default it would return an error, pushing down is disabled. This does mean however that those deployments with fielddata accessible are taxed.